.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: CI for RIF-Relay-Client
+on: [push]
+jobs:
+    linting:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '16.x'
+
+            - name: Install dependencies
+              run: npm install
+
+            - name: Check Linter bugs
+              run: npm run lint
+
+            - name: Fix Linter bugs
+              run: npm run lint:fix
+
+            - name: Check Codestyles errors
+              run: npm run prettier
+
+            - name: Fix Codestyles bugs
+              run: npm run prettier:fix
```

Comment:
I think that this task will never be executed because it's necessary when the previous one fails, so it's intended to be used locally by devs